
# Pipelines
* [A collection of high quality Nextflow pipelines](https://github.com/nf-core/)

### For Heatmap

* [iheatmapR](https://github.com/ropensci/iheatmapr)

# Co-expression network analysis

* [CEMiTool](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2053-1)

* [WGCNA](https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/)

* [LINC](https://bioconductor.org/packages/release/bioc/vignettes/LINC/inst/doc/LINC.html)

* [lionessR](https://bioconductor.org/packages/3.10/bioc/vignettes/lionessR/inst/doc/lionessR.html)

# Networks

* [STRING: functional protein association networks](https://string-db.org/cgi/download.pl?sessionId=LWFk7yB3M1MJ)

* [Markov Clustering](https://davetang.org/muse/2014/01/23/markov-clustering/)

# Gene-fusion detection

* [Clinker](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/gigascience/PAP/10.1093_gigascience_giy079/1/giy079_giga-d-18-00019_original_submission.pdf?Expires=2147483647&Signature=Pt5R93NcNmoccWLLWQv0rszneDqkFQN-z3QDrmMrfb3DRnSAmdvvmRbohvUBHHpsHIoOMMWWasd0IYU0jUPCUr8lqI472RW3NrRjQIsrh0AAMaKENa5A5kpCRvoNC8Hojcbv1a-yS019Fu4-XnCRUzlBbl4acBYgQ6Irl69j1-rFdWWwwCTrEtk-nBTcX6Kmp-himWrXfFTvfJukUWE4p~0NEmmTJX5PEmRSee59fkFrIhnWXyguwQsuytpTeM~5qWF5nQolfZ2pUWILI~fWnEbDOETkPcXYOgX0JsYcrjhRJscrxK7MG2c6GKWwJQa3xx6mNuLShmQ8QSpEmyDQSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)

* [Tools capable of detecting fusion genes](https://www.biostars.org/p/45986/)
* [FuSeq](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5156-1)

# Transcription factor prediction
* [BART](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty194/4956015)

# ATAC-Seq
* [ATAC-seq data analysis: from FASTQ to peaks](https://yiweiniu.github.io/blog/2019/03/ATAC-seq-data-analysis-from-FASTQ-to-peaks/)
* [From reads to insight: a hitchhikerâ€™s guide to ATAC-seq data analysis](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1929-3)

# Flow cytometry
* [CyTOF - Differential analysis of High-dimensional mass and flow cytometry](https://f1000research.com/articles/6-748/v4)

# DNA-methylation
* [DNA-methylation Analysis](https://f1000researchdata.s3.amazonaws.com/manuscripts/9514/98836561-348f-4784-8a5d-9b327062e500_8839_-_Jovana_Maksimovic.pdf?doi=10.12688/f1000research.8839.1)
* [Analysis of 450k DNA methylation data with minfi](https://kasperdanielhansen.github.io/genbioconductor/html/minfi.html)
* [decorate: Differential Epigenetic Correlation Test](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa067/5719017)

# Structural Variant Caller
* [Strelka2: fast and accurate calling of germline and somatic variants](https://www.nature.com/articles/s41592-018-0051-x)

# Geneset Enrichment Analysis
* [DESeq results to pathways with the fgsea package](https://stephenturner.github.io/deseq-to-fgsea/)
* [gerr: Gene set enrichment with regularized regression](https://github.com/TaoDFang/gerr)

# Mutational Signature Analysis
* [MuSiCa - Web app](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2234-y)
* [Helmsman: fast and efficient mutation signature analysis for massive sequencing datasets](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5264-y)
* [SMuRF: Portable and accurate ensemble prediction of somatic mutations](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz018/5288515)

# Single Cell RNA-Seq Analysis
* [A workflow for single cell RNA-seq data analysis](http://research.fhcrc.org/content/dam/stripe/sun/software/scRNAseq/scRNAseq.html)
* [A SingleCell RNASeq pre-processing snakemake workflow](https://github.com/Hoohm/dropSeqPipe)

# Super Enhancer Analysis
* [SEanalysis: a web tool for super-enhancer associated regulatory analysis](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz302/5480906)

# Integrating Data
* [Integrating ChIP-seq with other functional genomics data](https://academic.oup.com/bfg/article/17/2/104/4944665)

# Cancer Datasets portals and related R packages
* [TCGA Legacy Archive](https://portal.gdc.cancer.gov/legacy-archive/search/f)
* [TCGA Harmonized](https://portal.gdc.cancer.gov/)
* [Human Protein Atlas](https://www.proteinatlas.org/)
* [International Cancer Genome Consortium (ICGC)](https://dcc.icgc.org/)
* [cBioportal](http://www.cbioportal.org/)

* [TCGAbiolinks](https://bioconductor.org/packages/devel/bioc/vignettes/TCGAbiolinks/inst/doc/index.html)
* [GenomicDataCommons](https://bioconductor.org/packages/devel/bioc/vignettes/GenomicDataCommons/inst/doc/overview.html)
* [RTCGA](https://rtcga.github.io/RTCGA/)
* [FireBrowse](http://firebrowse.org/)
* [Xena Browser](https://xenabrowser.net/datapages/)
* [PAN Cancer Data GDC](https://gdc.cancer.gov/about-data/publications/pancanatlas)
* [Cancer Cell Line Encyclopedia](https://portals.broadinstitute.org/ccle)

# Different Tools and R packages
* [svist4get: a simple visualization tool for genomic tracks from sequencing experiments](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2706-8)
* [DegNorm: normalization of generalized transcript degradation improves accuracy in RNA-seq analysis](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1682-7)
* [CMScaller: an R package for consensus molecular subtyping of colorectal cancer pre-clinical models](https://www.nature.com/articles/s41598-017-16747-x)
* [Cancermine: Text-mined knowledgebase for drivers, oncogenes and tumor suppressors in cancer](https://github.com/jakelever/cancermine)
* [DeepLncRNA: Prediction of LncRNA Subcellular Localization with Deep Learning from Sequence Features](https://www.nature.com/articles/s41598-018-34708-w)
* [trackViewer: a Bioconductor package for interactive and integrative visualization of multi-omics data](https://www.nature.com/articles/s41592-019-0430-y)
* [pVACtools: a computational toolkit to identify and visualize cancer neoantigens]
* [RNASeqR : RNA-Seq analysis based on one independent variable](https://bioconductor.org/packages/release/bioc/vignettes/RNASeqR/inst/doc/RNASeqR.html)
* [LncADeep: An ab initio lncRNA identification and functional annotation tool based on deep learning](https://github.com/cyang235/LncADeep/)
* [circDeep: Deep learning approach for circular RNA classification from other long non-coding RNA](https://github.com/UofLBioinformatics/circDeep)
* [PDF Scraping in R with tabulizer](https://www.business-science.io/code-tools/2019/09/23/tabulizer-pdf-scraping.html)
* [The magick package: Advanced Image-Processing in R](https://cran.r-project.org/web/packages/magick/vignettes/intro.html)
* [Best practices for bioinformatic characterization of neoantigens for clinical utility](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-019-0666-2)
* [ViSEAGO:Visualisation, Semantic similarity, Enrichment Analysis of Gene Ontology.](https://bioconductor.org/packages/release/bioc/vignettes/ViSEAGO/inst/doc/ViSEAGO.html)

# Deep Learning in Bioinformatics
* [janggu](https://github.com/BIMSBbioinfo/janggu)


# Personal site designing on Github
* [github.io creation](http://jmcglone.com/guides/github-pages/#)


